### PR TITLE
Filipino Translation - 1109 translated words

### PR DESCRIPTION
**Pull request by smervs**
Translating this following two files :
console-and-shells/completion-shell.rst = 378 translated words
console-and-shells/upgrade-shell.rst = 38 translated words
console-and-shells/helpers.rst = 243 translated words
console-and-shells/i18n-shell.rst = 363 translated words
console-and-shells/orm-cache.rst = 87 translated words

Note : Code text, Filenames, Filepaths and computer generated text are not counted as translated words.

### CODE REVIEW

File: tl/bake/usage.rst [hunk 1]
```diff
@@ -1,19 +1,19 @@
-Code Generation with Bake
+Ang Paggenerate ng Code kasama ang Bake
```
Comment (mind-blast) [inline tl/bake/usage.rst]: Paggenerate to Pagbubuo

Comment (mind-blast) [reply tl/bake/usage.rst]: Code kasama ang Bake to Code na kasama ang Bake

File: tl/bake/usage.rst [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-If you have problems running the script, ensure that:
+Ang cake na console ay pinapatakbo gamit ang PHP na CLI (command line interface).
+Kung may mga problema ka sa pagpapatakbo ng script, siguraduhin na:
 
-#. You have the PHP CLI installed and that it has the proper modules enabled (eg: MySQL, intl).
-#. Users also might have issues if the database host is 'localhost' and should try '127.0.0.1' instead, as localhost can cause issues with PHP CLI.
-#. Depending on how your computer is configured, you may have to set execute rights on the cake bash script to call it using ``bin/cake bake``.
+#. Mayroon kang naka-install na PHP CLI at mayroon itong wastong mga modyul na pinagana (eg: MySQL, intl).
+#. Ang mga gumagamit ay maaaring magkaroon ng mga isyu kung ang host ng database ay 'localhost' at dapat subukan ang '127.0.0.1' sa halip, dahil ang localhost ay maaaring maging sanhi ng mga isyu sa CLI ng PHP.
+#. Dependi kung paano naka-configure ang iyong computer, maaari mong itakda ang mga karapatan sa pag-execute sa script ng cake na bash upang tawagin ito gamit ang ``bin/cake bake``.
```
Comment (mind-blast) [inline tl/bake/usage.rst]: Dependi to Depende

File: tl/bake/usage.rst [hunk 3]
```diff
@@ -115,11 +115,11 @@ available using the ``--help`` option::
                        Bake|Migrations)
     --verbose, -v      Enable verbose output.
 
-Bake Themes
+Mga Tema ng Bake
 ===========
 
-The theme option is common to all bake commands, and allows changing the bake
-template files used when baking. To create your own templates, see the
+Ang opsyon ng tema ay pangkaraniwan sa lahat ng mga command ng bake, at nagpapahintulot sa pagbabago ng bake
```
Comment (mind-blast) [inline tl/bake/usage.rst]: command  to utos

File: tl/bake/usage.rst [hunk 4]
```diff
@@ -115,11 +115,11 @@ available using the ``--help`` option::
...
     --verbose, -v      Enable verbose output.
 
-Bake Themes
+Mga Tema ng Bake
 ===========
 
-The theme option is common to all bake commands, and allows changing the bake
-template files used when baking. To create your own templates, see the
+Ang opsyon ng tema ay pangkaraniwan sa lahat ng mga command ng bake, at nagpapahintulot sa pagbabago ng bake 
+na template na mga file na ginagamit sa pagbi-bake. Para lumikha ng iyong sariling mga templates, tingnan ang
```
Comment (mind-blast) [inline tl/bake/usage.rst]: pagbi-bake to pag-bake

Comment (mind-blast) [reply tl/bake/usage.rst]: templates to template

File: tl/console-and-shells/completion-shell.rst [hunk 5]
```diff
@@ -1,91 +1,91 @@
...
+Pagkumpleto ng Shell
 ################
 
-Working with the console gives the developer a lot of possibilities but having
-to completely know and write those commands can be tedious. Especially when
-developing new shells where the commands differ per minute iteration. The
-Completion Shells aids in this matter by providing an API to write completion
-scripts for shells like bash, zsh, fish etc.
+Ang paggawa sa console ay nagbibigay sa developer ng maraming mga posibilidad ngunit ang pagkakaroon na
+upang lubos na malaman at maisulat ang mga command na iyon ay maaaring nakakapagod. Lalo na kapag
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command  to utos

File: tl/console-and-shells/completion-shell.rst [hunk 6]
```diff
@@ -1,91 +1,91 @@
...
-Completion Shells aids in this matter by providing an API to write completion
-scripts for shells like bash, zsh, fish etc.
+Ang paggawa sa console ay nagbibigay sa developer ng maraming mga posibilidad ngunit ang pagkakaroon na
+upang lubos na malaman at maisulat ang mga command na iyon ay maaaring nakakapagod. Lalo na kapag 
+nagbubuo ng mga bagong shell kung saan ang mga command ay naiiba sa bawat minuto ng pag-uulit. Ang 
+Pagkumpleto ng mga Shell ay tumutulong sa bagay na ito sa pamamagitan ng pagbibigay ng isang API upang magsulat ng kumpletong 
+mga script para sa mga shell tulad ng bash, zsh, fish atbp.
 
-Sub Commands
+Mga Sub na Command
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: Command to Utos

File: tl/console-and-shells/completion-shell.rst [hunk 7]
```diff
@@ -1,91 +1,91 @@
...
 ============
 
-The Completion Shell consists of a number of sub commands to assist the
-developer creating its completion script. Each for a different step in the
-autocompletion process.
+Ang Pagkumpleto ng Shell ay binubuo ng isang bilang ng mga sub na command upang tulungan ang
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 8]
```diff
@@ -1,91 +1,91 @@
...
+autocompletion na proseso.
 
-Commands
+Mga Command
 --------
 
-For the first step commands outputs the available Shell Commands, including
-plugin name when applicable. (All returned possibilities, for this and the other
-sub commands, are separated by a space.) For example::
+Para sa unang hakbang ng mga command ay nag-output ng magagamit na mga Command ng Shell, kasama ang
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command  to utos

File: tl/console-and-shells/completion-shell.rst [hunk 9]
```diff
@@ -1,91 +1,91 @@
...
-Commands
+Mga Command
 --------
 
-For the first step commands outputs the available Shell Commands, including
-plugin name when applicable. (All returned possibilities, for this and the other
-sub commands, are separated by a space.) For example::
+Para sa unang hakbang ng mga command ay nag-output ng magagamit na mga Command ng Shell, kasama ang 
+pangalan ng plugin kapag naaangkop. (Lahat ng ibinalik ng mga posibilidad, para dito at sa iba pang 
+mga sub na command, ay pinaghihiwalay ng isang puwang.) Halimbawa::
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 10]
```diff
@@ -1,91 +1,91 @@
...
     bin/cake Completion commands
 
-Returns::
+Ibabalik::
 
     acl api bake command_list completion console i18n schema server test testsuite upgrade
 
-Your completion script can select the relevant commands from that list to
-continue with. (For this and the following sub commands.)
+Ang iyong script ng pagkumpleto ay maaaring pumili ng may katuturang mga command mula sa listahang iyon para
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 11]
```diff
@@ -1,91 +1,91 @@
...
+magpagtuloy. (Para dito at sa sumusunod na mga sub na command.)
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 12]
```diff
@@ -1,91 +1,91 @@
...
 
-subCommands
+mga subCommand
 -----------
 
-Once the preferred command has been chosen subCommands comes in as the second
-step and outputs the possible sub command for the given shell command. For
-example::
+Kapang ang preferred na command ay napili na ang mga subCommand dadating bilang ang pangalawang
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 13]
```diff
@@ -1,91 +1,91 @@
...
 
-subCommands
+mga subCommand
 -----------
 
-Once the preferred command has been chosen subCommands comes in as the second
-step and outputs the possible sub command for the given shell command. For
-example::
+Kapang ang preferred na command ay napili na ang mga subCommand dadating bilang ang pangalawang 
+hakbang at i-output ang posibleng sub na command para sa ibinigay na command ng shell.
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 14]
```diff
@@ -1,91 +1,91 @@
...
 
-.. note::
+.. tandaan::
 
-    If you are using MacOS X, you can install the **bash-completion** library
-    using **homebrew** with the command ``brew install bash-completion``.
-    The target directory for the **cake** file will be
+    Kung ikaw ay gumagamit ng MacOS X, maaari mong i-install ang **bash-completion** na library
+    gamit ang **homebrew** na may command na ``brew install bash-completion``.
+    Ang target na directory para sa **cake** na file ay magiging
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: directory  to direktoryo

File: tl/console-and-shells/completion-shell.rst [hunk 15]
```diff
@@ -139,26 +139,26 @@ in order to get autocompletion when using the CakePHP console::
...
     complete -F _cake cake bin/cake
 
-Using autocompletion
+Paggamit ng autocompletion
 ====================
 
-Once enabled, the autocompletion can be used the same way than for other
-built-in commands, using the **TAB** key.
-Three type of autocompletion are provided. The following output are from a fresh CakePHP install.
+Kapag na-enable, ang autocompletion ay maaaring gamitin sa parehong paraan kaysa sa ibang
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: na-enable to napagana

File: tl/console-and-shells/completion-shell.rst [hunk 16]
```diff
@@ -139,26 +139,26 @@ in order to get autocompletion when using the CakePHP console::
...
 
-Using autocompletion
+Paggamit ng autocompletion
 ====================
 
-Once enabled, the autocompletion can be used the same way than for other
-built-in commands, using the **TAB** key.
-Three type of autocompletion are provided. The following output are from a fresh CakePHP install.
+Kapag na-enable, ang autocompletion ay maaaring gamitin sa parehong paraan kaysa sa ibang 
+built-in na mga command, gamit ang **TAB** na key.
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/completion-shell.rst [hunk 17]
```diff
@@ -139,26 +139,26 @@ in order to get autocompletion when using the CakePHP console::
...
+Tatlong uri ng autocompletion ang ibibigay. Ang sumusunod na output ay mula sa isang sariwang pag-install ng CakePHP.
 
-Commands
+Mga Command
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: Command to Utos

File: tl/console-and-shells/completion-shell.rst [hunk 18]
```diff
@@ -139,26 +139,26 @@ in order to get autocompletion when using the CakePHP console::
...
 --------
 
-Sample output for commands autocompletion::
+Sample na output para sa autocompletion ng mga command::
```
Comment (mind-blast) [inline tl/console-and-shells/completion-shell.rst]: command to utos

File: tl/console-and-shells/i18n-shell.rst [hunk 19]
```diff
@@ -1,73 +1,73 @@
-I18N Shell
+I18N na Shell
 ##########
 
-The i18n features of CakePHP use `po files <http://en.wikipedia.org/wiki/GNU_gettext>`_
-as their translation source. PO files integrate with commonly used translation tools
-like `Poedit <http://www.poedit.net/>`_.
+Ang i18n na mga feature ng CakePHP ay gumagamit ng `po files <http://en.wikipedia.org/wiki/GNU_gettext>`_
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: feature to tampok

File: tl/console-and-shells/i18n-shell.rst [hunk 20]
```diff
@@ -1,73 +1,73 @@
...
 
-Generating POT Files
+Pagbubuo ng mga File ng POT
 ====================
 
-POT files can be generated for an existing application using the ``extract``
-command. This command will scan your entire application for ``__()`` style
-function calls, and extract the message string. Each unique string in your
-application will be combined into a single POT file::
+Ang mga POT files ay maaaring mabuo para sa umiiral na aplikasyon gamit ang ``extract``
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: files  to na mga file

File: tl/console-and-shells/i18n-shell.rst [hunk 21]
```diff
@@ -1,73 +1,73 @@
...
+na command. Ang command na ito ay i-scan ang iyong buong aplikasyon para sa ``__()`` na estilo
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: command to utos

File: tl/console-and-shells/i18n-shell.rst [hunk 22]
```diff
@@ -1,73 +1,73 @@
...
+ng pagtawag ng function, at i-extract ang string ng mensahe. Ang bawat natatanging string sa iyong 
+aplikasyon ay sasamahin sa isang file ng POT::
 
     bin/cake i18n extract
 
-The above will run the extraction shell. The result of this command will be the
-file **src/Locale/default.pot**. You use the pot file as a template for creating
-po files. If you are manually creating po files from the pot file, be sure to
-correctly set the ``Plural-Forms`` header line.
+Ang nasa itaas ay ipatatakbo ang shell ng pagkuha. Ang resulta ng command na ito ay ang
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: command to utos

File: tl/console-and-shells/i18n-shell.rst [hunk 23]
```diff
@@ -1,73 +1,73 @@
...
+Pag-extract mula sa maramihang mga folder nang sabay-sabay
 ----------------------------------------
 
-Sometimes, you might need to extract strings from more than one directory of
-your application. For instance, if you are defining some strings in the
-``config`` directory of your application, you probably want to extract strings
-from this directory as well as from the ``src`` directory. You can do it by
-using the ``--paths`` option. It takes a comma-separated list of absolute paths
-to extract::
+Minsan, maaaring kailanganin mong kunin ang mga string mula sa higit sa isang directory ng
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: directory  to direktoryo

File: tl/console-and-shells/i18n-shell.rst [hunk 24]
```diff
@@ -1,73 +1,73 @@
...
 
-Sometimes, you might need to extract strings from more than one directory of
-your application. For instance, if you are defining some strings in the
-``config`` directory of your application, you probably want to extract strings
-from this directory as well as from the ``src`` directory. You can do it by
-using the ``--paths`` option. It takes a comma-separated list of absolute paths
-to extract::
+Minsan, maaaring kailanganin mong kunin ang mga string mula sa higit sa isang directory ng 
+iyong aplikasyon. Halimbawa, kung tinutukoy mo ang ilang mga string sa 
+``config`` na directory ng iyong aplikasyon, baka gusto mong kunin ang mga string
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: directory  to direktoryo

File: tl/console-and-shells/i18n-shell.rst [hunk 25]
```diff
@@ -1,73 +1,73 @@
...
-Sometimes, you might need to extract strings from more than one directory of
-your application. For instance, if you are defining some strings in the
-``config`` directory of your application, you probably want to extract strings
-from this directory as well as from the ``src`` directory. You can do it by
-using the ``--paths`` option. It takes a comma-separated list of absolute paths
-to extract::
+Minsan, maaaring kailanganin mong kunin ang mga string mula sa higit sa isang directory ng 
+iyong aplikasyon. Halimbawa, kung tinutukoy mo ang ilang mga string sa 
+``config`` na directory ng iyong aplikasyon, baka gusto mong kunin ang mga string 
+mula sa directory na ito pati na rin mula sa ``src`` na directory. Magagawa mo ito sa pamamagitan ng
```
Comment (mind-blast) [inline tl/console-and-shells/i18n-shell.rst]: directory to direktoryo

File: tl/console-and-shells/orm-cache.rst [hunk 26]
```diff
@@ -1,21 +1,21 @@
-ORM Cache Shell
+ORM na Shell ng Cache
 ###############
 
-The OrmCacheShell provides a simple CLI tool for managing your application's
-metadata caches. In deployment situations it is helpful to rebuild the metadata
-cache in-place without clearing the existing cache data. You can do this by
-running::
+Ang OrmCacheShell ay nagbibigay ng isang simpleng CLI na tool para sa pamamahala ng iyong aplikasyon na
```
Comment (mind-blast) [inline tl/console-and-shells/orm-cache.rst]: tool  to kasangkapan

File: tl/bake/usage.rst [hunk 27]
```diff
@@ -1,19 +1,19 @@
-Ang Paggenerate ng Code kasama ang Bake
+Code Generation with Bake
```
Comment (mind-blast) [inline tl/bake/usage.rst]: Please check all usage.rst many are not translated.

**Comment by mind-blast** [comment]
Please check on usage.rst there are items that are not translated. Please check all suggested words also.

**Comment by smervs** [comment]
usage.rst is not included on this PR. please see this link: https://github.com/cakephp-tl/docs/pull/1 for usage.rst

**Comment by smervs** [comment]
Done updating the files.

File: tl/console-and-shells/completion-shell.rst [hunk 28]
```diff
@@ -1,91 +1,91 @@
...
+Pagkumpleto ng Shell
 ################
 
-Working with the console gives the developer a lot of possibilities but having
-to completely know and write those commands can be tedious. Especially when
-developing new shells where the commands differ per minute iteration. The
-Completion Shells aids in this matter by providing an API to write completion
-scripts for shells like bash, zsh, fish etc.
+Ang paggawa sa console ay nagbibigay sa developer ng maraming mga posibilidad ngunit ang pagkakaroon na
+upang lubos na malaman at maisulat ang mga utos na iyon ay maaaring nakakapagod. Lalo na kapag
```
Comment (severinolorillajr) [inline tl/console-and-shells/completion-shell.rst]: >ngunit para lubosang malaman at maisulat

File: tl/console-and-shells/completion-shell.rst [hunk 29]
```diff
@@ -1,91 +1,91 @@
...
 
-The Completion Shell consists of a number of sub commands to assist the
-developer creating its completion script. Each for a different step in the
-autocompletion process.
+Ang Pagkumpleto ng Shell ay binubuo ng isang bilang ng mga sub na utos upang tulungan ang 
+developer na lumikha ng kumpletong script. Bawat isa para sa ibang hakbang sa 
+autocompletion na proseso.
 
-Commands
+Mga Command
```
Comment (severinolorillajr) [inline tl/console-and-shells/completion-shell.rst]: Mga Utos

File: tl/console-and-shells/completion-shell.rst [hunk 30]
```diff
@@ -1,91 +1,91 @@
...
+magpagtuloy. (Para dito at sa sumusunod na mga sub na utos.)
 
-subCommands
+mga subCommand
 -----------
 
-Once the preferred command has been chosen subCommands comes in as the second
-step and outputs the possible sub command for the given shell command. For
-example::
+Kapang ang preferred na utos ay napili na ang mga subCommand dadating bilang ang pangalawang
```
Comment (severinolorillajr) [inline tl/console-and-shells/completion-shell.rst]: Kapag ang nagustuhang utos ay napili na, ang mga subCommand ay papasok bilang pangalawang

File: tl/console-and-shells/completion-shell.rst [hunk 31]
```diff
@@ -1,91 +1,91 @@
...
     controller db_config fixture model plugin project test view
 
-options
+mga opsyon
 -------
 
-As the third and final options outputs options for the given (sub) command as
-set in getOptionParser. (Including the default options inherited from Shell.)
-For example::
+Habang ang pangatlo at pangwakas ay mag-output ng mga opsyon para sa ibinigay na (sub) utos bilang
```
Comment (severinolorillajr) [inline tl/console-and-shells/completion-shell.rst]: pangwakas na mga opsyon ay nag-output ng mga pagpipilian para sa

File: tl/console-and-shells/completion-shell.rst [hunk 32]
```diff
@@ -1,91 +1,91 @@
...
-First, make sure the **bash-completion** library is installed. If not, you do it
-with the following command::
+Una, siguraduhin na ang **bash-completion** na library ay naka-install. Kung hindi, gagawin mo ito 
+kasama ang sumusunod na utos::
 
     apt-get install bash-completion
 
-Create a file named **cake** in **/etc/bash_completion.d/** and put the
-:ref:`bash-completion-file-content` inside it.
+Lumikha ng file na pinangalanang **cake** in **/etc/bash_completion.d/** at ilagay ang
```
Comment (severinolorillajr) [inline tl/console-and-shells/completion-shell.rst]: sa

File: tl/console-and-shells/helpers.rst [hunk 33]
```diff
@@ -1,32 +1,32 @@
-Shell Helpers
+Mga Helper ng Shell
 #############
 
 .. versionadded:: 3.1
     Shell Helpers were added in 3.1.0
 
-Shell Helpers let you package up complex output generation code. Shell
-Helpers can be accessed and used from any shell or task::
+Ang mga Helper ng Shell ay hinahayaan kang i-package ang kumplikadong output ng generation na code. Ang
```
Comment (severinolorillajr) [inline tl/console-and-shells/helpers.rst:7]: paglikha ng output na code.

File: tl/console-and-shells/i18n-shell.rst [hunk 34]
```diff
@@ -1,73 +1,73 @@
-I18N Shell
+I18N na Shell
 ##########
 
-The i18n features of CakePHP use `po files <http://en.wikipedia.org/wiki/GNU_gettext>`_
-as their translation source. PO files integrate with commonly used translation tools
-like `Poedit <http://www.poedit.net/>`_.
+Ang i18n na mga tampok ng CakePHP ay gumagamit ng `po files <http://en.wikipedia.org/wiki/GNU_gettext>`_
+bilang kanilang pinagmulan ng pagsasalin. Angg PO na mga file ay i-integrate sa karaniwang ginagamit na mga tool sa pagsasalin
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst:5]: Angg

Comment (severinolorillajr) [reply tl/console-and-shells/i18n-shell.rst:5]: tool to kagamitan

File: tl/console-and-shells/i18n-shell.rst [hunk 35]
```diff
@@ -1,73 +1,73 @@
...
+bilang kanilang pinagmulan ng pagsasalin. Angg PO na mga file ay i-integrate sa karaniwang ginagamit na mga tool sa pagsasalin 
+tulad ng `Poedit <http://www.poedit.net/>`_.
 
-The i18n shell provides a quick and easy way to generate po template files.
-These templates files can then be given to translators so they can translate the
-strings in your application. Once you have translations done, pot files can be
-merged with existing translations to help update your translations.
+Ang i18n na shell ay nagbibigay ng isang mabilis at madaling paraan upang makabuo ng mga file ng template ng po.
+Ang mga file ng mga template na ito ay maaaring ibigay sa mga tagasalin upang maisalin nila ang 
+mga string na iyong aplikasyon. Kapag meron ka nang natapos na pagsasalin, ang mga file ng pot ay maaaring
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst:10]: mga string sa iyong aplikasyon.

File: tl/console-and-shells/i18n-shell.rst [hunk 36]
```diff
@@ -1,73 +1,73 @@
...
-Generating POT Files
+Pagbubuo ng mga File ng POT
 ====================
 
-POT files can be generated for an existing application using the ``extract``
-command. This command will scan your entire application for ``__()`` style
-function calls, and extract the message string. Each unique string in your
-application will be combined into a single POT file::
+Ang mga POT na mga file ay maaaring mabuo para sa umiiral na aplikasyon gamit ang ``extract``
+na utos. Ang utos na ito ay i-scan ang iyong buong aplikasyon para sa ``__()`` na estilo
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst:17]: mag-scan sa itong buong aplikasyon

File: tl/console-and-shells/i18n-shell.rst [hunk 37]
```diff
@@ -1,73 +1,73 @@
...
 
     bin/cake i18n extract --paths /var/www/app/config,/var/www/app/src
 
-Excluding Folders
+Pag-exclude ng mga Folder
 -----------------
 
-You can pass a comma separated list of folders that you wish to be excluded.
-Any path containing a path segment with the provided values will be ignored::
+Maaari kang pumasa ng listahan ng isang pinaghiwalay ng comma na mga folder na nais mong i-exclude.
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst:52]: magpasa ng

File: tl/console-and-shells/i18n-shell.rst [hunk 38]
```diff
@@ -1,73 +1,73 @@
...
+Anumang path na naglalaman ng isang segment ng path na may ibinigay na mga halaga ay hindi papansinin::
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst:53]: Anumang landas .... bahagi ng landas nang naibigay na mga halaga ay isasawalang-bahala

File: tl/console-and-shells/i18n-shell.rst [hunk 39]
```diff
@@ -1,73 +1,73 @@
...
 
     bin/cake i18n extract --exclude Test,Vendor
 
-Skipping Overwrite Warnings for Existing POT Files
+Pag-skip ng mga Babala ng pag-overwrite para sa Umiiral na mga File ng POT
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst]: Paglaktaw sa mga Pag-Overwrite na Babala para sa Umiiral na POT na mga File

File: tl/console-and-shells/i18n-shell.rst [hunk 40]
```diff
@@ -1,73 +1,73 @@
...
 --------------------------------------------------
 
-By adding ``--overwrite``, the shell script will no longer warn you if a POT
-file already exists and will overwrite by default::
+Sa pagdaragdag ng ``--overwrite``, ang script ng shell ay hindi ka na babalaan kung ang file ng POT 
+ay umiiral na at i-overwrite bilang default::
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst]: mag-overwrite

File: tl/console-and-shells/i18n-shell.rst [hunk 41]
```diff
@@ -1,73 +1,73 @@
...
-Extracting Messages from the CakePHP Core Libraries
+Pag-extract ng mga Mensahe mula sa Core na mga Library ng CakePHP
 ---------------------------------------------------
 
-By default, the extract shell script will ask you if you like to extract
-the messages used in the CakePHP core libraries. Set ``--extract-core`` to yes
-or no to set the default behavior::
+Bilang default, ang shell na script ng pag-extract ay tatanungin ka kung gusto mong i-extract 
+ang mga mensahe na ginamit sa core na mga library ng CakePHP. I-set ang ``--extract-core`` sa oo 
+o hindi upang i-set ang default na pag-uugali::
```
Comment (severinolorillajr) [inline tl/console-and-shells/i18n-shell.rst]: itakda ang default na gawi

File: tl/console-and-shells/orm-cache.rst [hunk 42]
```diff
@@ -1,21 +1,21 @@
-ORM Cache Shell
+ORM na Shell ng Cache
 ###############
 
-The OrmCacheShell provides a simple CLI tool for managing your application's
-metadata caches. In deployment situations it is helpful to rebuild the metadata
-cache in-place without clearing the existing cache data. You can do this by
-running::
+Ang OrmCacheShell ay nagbibigay ng isang simpleng CLI na kasangkapan para sa pamamahala ng iyong aplikasyon na 
+mga cache ng metadata. Sa mga sitwasyon ng pagdeploy ito ay makakatulong na muling itayo ang cache ng metadata
```
Comment (severinolorillajr) [inline tl/console-and-shells/orm-cache.rst]: pagdeploy, ito ay

File: tl/console-and-shells/orm-cache.rst [hunk 43]
```diff
@@ -1,21 +1,21 @@
...
+ORM na Shell ng Cache
 ###############
 
-The OrmCacheShell provides a simple CLI tool for managing your application's
-metadata caches. In deployment situations it is helpful to rebuild the metadata
-cache in-place without clearing the existing cache data. You can do this by
-running::
+Ang OrmCacheShell ay nagbibigay ng isang simpleng CLI na kasangkapan para sa pamamahala ng iyong aplikasyon na 
+mga cache ng metadata. Sa mga sitwasyon ng pagdeploy ito ay makakatulong na muling itayo ang cache ng metadata 
+sa parehong lugar na hindi i-clear ang umiiral na data ng cache. Maaari mong gawin ito sa pamamagitan ng
```
Comment (severinolorillajr) [inline tl/console-and-shells/orm-cache.rst]: hindi nangangailangan tanggalin

**Comment by smervs** [comment]
Done updating the files. Thank you for the proofread.

**Pull request merged**